analysis/plots.py

```diff
@@ -69,6 +69,8 @@ def checkColumnDict(dic, key):
             convert_binary(df, 'learning_disability', 'Record of learning disability', 'No record of learning disability')
         if (breakdownby == "nhome"):
             convert_binary(df, 'nhome', 'Record of individual living at a nursing home', 'No record of individual living at a nursing home')
+        if (breakdownby == "addictivemeds_last12m"):
```

Comment:
It's become a little messy as column names don't match filenames for the high risk meds. That's a learning point for me. I've added a workaround (see line 63) which converts filename to column name as a new variable breakdownbycol
```suggestion
        if (breakdownbycol == "addictivemeds_last12m"):
```